src/sidebar.js

```diff
@@ -94,13 +89,18 @@ class Sidebar extends React.Component {
     // filter out if a user starts swiping with a second finger
     if (!this.isTouching()) {
       let touch = ev.targetTouches[0];
+      let tcX = touch.clientX;
+      if (this.props.pullRight) {
+        tcX = this.state.sidebarWidth - tcX;
```

Comment:
Sorry, I meant the methods `inCancelDistanceOnScroll()` and `touchSidebarWidth()`. They are actually using the coordinates and should be made pullRight aware.